README.adoc

```diff
@@ -0,0 +1,34 @@
+= Openshift Pipelines Documentation
+
+WARNING: This documentation repo has been deprecated. Please visit the official OpenShift Container Platform (OCP) documentation at link:https://docs.openshift.com/container-platform/4.4/welcome/index.html[OCP 4.4 Documentation]
```

Comment:
Let's link to the exact pipeline docs

Reply:
https://docs.openshift.com/container-platform/latest/pipelines/understanding-openshift-pipelines.html

Reply:
OCP 4.4 Documentation change this to OpenShift Pipelines documentation

Reply:
(latest + an exact pipelines docs (understanding pipelins)